test/rules/lines_longer_than_80_chars.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `pub run test -N lines_longer_than_80_chars`
+
+var short = 'This is a short line'; // OK
+var line80 =
+    '                                                                   '; // OK
+var line81 =
+    '                                                                  '; // LINT
+var long =
+    'This line is a long, very long, very very long, very very very long, very very very very long'; // LINT
+
+var uriIsOk =
```

Comment:
We should have some tests for file paths (both Posix and Windows style).

And if we're going to use a pattern, it would be good to have some additional URIs with additional structure to ensure that the pattern covers all of the cases.

Reply:
Done